components/style/values/specified/box.rs

```diff
@@ -130,3 +131,63 @@ define_css_keyword_enum! { OverflowClipBox:
     "content-box" => ContentBox,
 }
 add_impls_for_keyword_enum!(OverflowClipBox);
+
+#[derive(Clone, Debug, MallocSizeOf, PartialEq, ToComputedValue)]
+/// Provides a rendering hint to the user agent,
+/// stating what kinds of changes the author expects
+/// to perform on the element
+///
+/// <https://drafts.csswg.org/css-will-change/#will-change>
+pub enum WillChange {
+    /// Expresses no particular intent
+    Auto,
+    /// scroll-position | contents | <custom-ident>
+    AnimateableFeatures(Vec<CustomIdent>),
```

Comment:
You may want to also use `Box<[CustomIdent]>` to save some memory, but again, completely optional :)

Reply:
Let me try to use `Box` again!